prow/config/jobs/metal3-io/metal3-dev-env.yaml

```diff
@@ -66,6 +74,10 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  - name: metal3-centos-e2e-integration-test-release-1-10
+    agent: jenkins
+    always_run: false
+    optional: false
   - name: metal3-centos-e2e-integration-test-release-1-9
     agent: jenkins
     always_run: false
```

Comment:
Optional: true on the next line for release 1.9, otherwise we'll have 3 test suites as mandatory